src/notebooks/graphql/queries.graphql

```diff
@@ -0,0 +1,3 @@
+query notebooksUrl{
```

Comment:
Wouldn't that be more NotebooksPage query ?

Reply:
Yes, will update it